php/libraries/Utility.class.inc

```diff
@@ -132,12 +132,15 @@ class Utility
         $DB      = $factory->database();
 
         // get the list of study sites - to be replaced by the Site object
-        $query = "SELECT CenterID, Name FROM psc ";
+        $query  = "SELECT CenterID, Name FROM psc ";
+        $query .= "WHERE ";
```

Comment:
In this form the SQL query will always fail if `Study_site=false` and `dcc=true` because the WHERE clause is incomplete.

Reply:
Latest commit adds if statement around the WHERE clause, checking if any of the variables' values match the conditions needed for it.